ui/selectors/selectors.js

```diff
@@ -1916,6 +1916,35 @@ export const getCurrentNetwork = createDeepEqualSelector(
   },
 );
 
+export const getSelectedNetwork = createDeepEqualSelector(
```

Comment:
I realize that we already have `getProviderConfig`, but unfortunately that doesn't give me what I need. I need both the `selectedNetworkClientId` and the chain ID that corresponds to the selected network. I guess I could have reused `getCurrentChainId` in this case, but I'm already making use of `networkConfigurationsByChainId` and it gives me everything I need.